rest/route/project.go

```diff
@@ -328,6 +328,9 @@ func (h *projectIDPatchHandler) Parse(ctx context.Context, r *http.Request) erro
 
 // Run updates a project by name.
 func (h *projectIDPatchHandler) Run(ctx context.Context) gimlet.Responder {
+	if h.newProjectRef.IsHidden() {
```

Comment:
Should this also apply to other patch creation routes like submitting commit queue patches and such? Or do we assume that users wouldn't try submitting work for a hidden project, and this is just a spot check?

Reply:
Submitting commit queue should hit an issue with the commit queue being disabled but yeah generally this is just a spot check.